docs/man/man1/kubectl-annotate.1

```diff
@@ -63,8 +63,8 @@ resourcequotas (quota) or secrets.
 
 .PP
 \fB\-t\fP, \fB\-\-template\fP=""
-    Template string or path to template file to use when \-o=template or \-o=templatefile.  The template format is golang templates [
-\[la]http://golang.org/pkg/text/template/#pkg-overview\[ra]]
+    Template string or path to template file to use when \-o=template or \-o=templatefile.  \-t is DEPRECATED to support \-\-tty.  The template format is golang templates [
```

Comment:
I wouldn't mention that we plan to use -t for --tty -- that may just confuse users.